flight/Modules/Stabilization/stabilization.c

```diff
@@ -863,6 +867,10 @@ static void stabilizationTask(void* parameters)
 				stabDesired.ThrustMode, &flightStatus,
 				airframe_type, stabDesired.Thrust);
 
+		actuatorDesired.FlipOverThrustMode =
```

Comment:
This is the real work in stabilization (not much)-- just signal to actuator what we want.